### PR TITLE
Compress packages updated

### PR DESCRIPTION
**Pull request by heitbaum**
Update the following compres packages to current:

libarchive/package.mk 3.5.0
lz4/package.mk 1.9.3
minizip/package.mk 2.10.4
xz/package.mk 5.2.5

**Note: fixed minizip - it hadn't been compiling. miniunz_exec and minizip_exec were not valid targets, even when BUILD_TEST was corrected to MZ_BUILD_TEST

### CODE REVIEW

**Review by CvH — APPROVED** [review]
can we change to xz ? saves some MB at download

File: packages/compress/libarchive/package.mk [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libarchive"
-PKG_VERSION="3.4.2"
-PKG_SHA256="b60d58d12632ecf1e8fad7316dc82c6b9738a35625746b47ecdcaf4aed176176"
+PKG_VERSION="3.5.0"
+PKG_SHA256="fc4bc301188376adc18780d35602454cc8df6396e1b040fbcbb0d4c0469faf54"
 PKG_LICENSE="GPL"
 PKG_SITE="https://www.libarchive.org"
 PKG_URL="https://www.libarchive.org/downloads/$PKG_NAME-$PKG_VERSION.tar.gz"
```
[the remark this answers is not in the document]
Comment (heitbaum) [reply packages/compress/libarchive/package.mk]: Fixed up :-)

File: packages/compress/libarchive/package.mk [hunk 2]
```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libarchive"
-PKG_VERSION="3.4.2"
-PKG_SHA256="b60d58d12632ecf1e8fad7316dc82c6b9738a35625746b47ecdcaf4aed176176"
+PKG_VERSION="3.5.0"
+PKG_SHA256="fc4bc301188376adc18780d35602454cc8df6396e1b040fbcbb0d4c0469faf54"
```
[the remark this answers is not in the document]
Comment (heitbaum) [reply packages/compress/libarchive/package.mk]: Fixed up :-)

**Comment by CvH** [comment]
anyone an idea if we need minizip ?
https://github.com/LibreELEC/LibreELEC.tv/pull/2086 isn't helping either
looks like we could drop it
```
$ git grep minizip
packages/compress/minizip/package.mk:PKG_NAME="minizip"
packages/compress/minizip/package.mk:PKG_SITE="https://github.com/nmoinvaz/minizip"
packages/compress/minizip/package.mk:PKG_URL="https://github.com/nmoinvaz/minizip/archive/$PKG_VERSION.tar.gz"
packages/compress/minizip/package.mk:  cp -v minizip_exec $SYSROOT_PREFIX/usr/bin/minizip
```

**Comment by chewitt** [comment]
The minizip package came into existence with #2086 but I have no idea what it was used for @lrusak @LongChair ? .. defo looks like it's unused so would be a candidate for being dropped.

**Comment by lrusak** [comment]
> The minizip package came into existence with #2086 but I have no idea what it was used for @lrusak @LongChair ? .. defo looks like it's unused so would be a candidate for being dropped.

Yea it can probably be removed. It was added for PHT which I think is defunct now.

**Comment by heitbaum** [comment]
minizip dropped from this PR.
minizip 2.10.4 commit backed up in - https://github.com/LibreELEC/LibreELEC.tv/commit/bdcd47fce72d7374cd7f5875140c923b1b1f4cb3

**Pull request merged**